google-codelab-index/_categories.scss

```diff
@@ -0,0 +1,98 @@
+
+/**
+ * Copyright 2015 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Common colors used in more than one category. */
+$appsBackgroundColor:     #D93025;
+$unityBackgroundColor:    #5F6368;
+$webBackgroundColor:      #1A73E8;
+/* Common icons used in more than one category. */
+$webIcon: 'web.svg';
+
+/* Mixins. */
+@mixin colorize-card($name, $color, $icon:false) {
+    .#{$name}-bg {
+      background-color: $color;
+    }
+
+    @if $icon {
+      .#{$name}-icon {
+        background-image: url(//codelabs.developers.google.com/images/icons/#{$icon});
+      }
+    } @else {
+      .#{$name}-icon {
```

Comment:
Or the if/else could be inside a single `.#{$name}-icon` but I don't really care. You probably had other reasons for this.
Ah, maybe SCSS doesn't allow `@if` statements within a CSS class definition.

Reply:
It totally can!